From e4146a91857886ee329f3e7a91c77e9ba27812cb Mon Sep 17 00:00:00 2001
From: Bruce Ashfield <bruce.ashfield@windriver.com>
Date: Fri, 28 May 2010 15:52:49 -0400
Subject: [PATCH 04/15] ck: cleanup swapiness tunable

Swappiness the tunable lies. It doesn't respect swappiness because it alters
the value when we're more than lightly loaded in the vm. Change it to -really-
mean swappiness unless we're about to go out of memory.

-ck

Signed-off-by: Bruce Ashfield <bruce.ashfield@windriver.com>
---
 mm/vmscan.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/mm/vmscan.c b/mm/vmscan.c
index 913516c..954e1f3 100644
--- a/mm/vmscan.c
+++ b/mm/vmscan.c
@@ -126,7 +126,7 @@ struct scan_control {
 /*
  * From 0 .. 100.  Higher means more swappy.
  */
-int vm_swappiness = 60;
+int vm_swappiness;
 long vm_total_pages;	/* The total number of pages which the VM controls */
 
 static LIST_HEAD(shrinker_list);
-- 
1.6.5.2

